prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -366,6 +366,50 @@ presubmits: # runs on PRs
             effect: NoSchedule
         nodeSelector:
             dedicated: "high-cpu"
+    - name: pre-main-cluster-users-integration-minikube
+      annotations:
+        pipeline.trigger: "pr-submit"
```

Comment:
Can't we provide more annotations? I think these may be useful to quickly determine what does the job do.

Reply:
I've manually added some annotations similar to the job above. ptal